src/ygg_nice_ipv6.go

```diff
@@ -109,9 +110,13 @@ func main() {
 		specialBytes, err := hex.DecodeString(specialBytesString)
 		if err != nil {
 			// handle error
+			fmt.Println("Check you correctly entered special bytes parameter")
 			fmt.Println(err)
 			os.Exit(2)
 		}
+		fmt.Printf("Target: find %v %v times in address\n\n", specialBytesString, matchesString)
+		n, err := file.WriteString(fmt.Sprintf("Target: find %v %v times in address\n\n", specialBytesString, matchesString))
+		checkWithN(err, n)
```

Comment:
Will print target to file and on screen, for mirror blocks target I forgot to add this